src/app/modules/feedback/feedback.util.ts

```diff
@@ -0,0 +1,63 @@
+import { StatusCodes } from 'http-status-codes'
+
+import { MapRouteError } from '../../../types'
+import { createLoggerWithLabel } from '../../config/logger'
+import { ApplicationError, DatabaseError } from '../core/core.errors'
+import * as FormErrors from '../form/form.errors'
+
+import {
+  DuplicateFeedbackSubmissionError,
+  InvalidSubmissionIdError,
+} from './feedback.error'
+
+const logger = createLoggerWithLabel(module)
+
+export const mapRouteError: MapRouteError = (
+  error: ApplicationError,
+  coreErrorMessage?: string,
+) => {
+  switch (error.constructor) {
+    case FormErrors.FormNotFoundError:
+      return {
+        statusCode: StatusCodes.NOT_FOUND,
+        errorMessage: error.message,
+      }
+    case FormErrors.FormDeletedError:
+      return {
+        statusCode: StatusCodes.GONE,
+        errorMessage: error.message,
```

Comment:
similar to the previous comment, would be good to go through all these error messages and make sure that they make sense in the context of form feedback.